package.json

```diff
@@ -140,13 +139,15 @@
     "stylelint-config-standard": "^34.0.0",
     "stylelint-prettier": "^4.0.2",
     "svg-symbols": "^1.0.5",
+    "tracked-built-ins": "^3.3.0",
     "webpack": "^5.88.2"
   },
   "optionalDependencies": {
     "ember-power-select": "2.x || 3.x || 4.x || 5.x || 6.x || 7.x"
   },
   "peerDependencies": {
-    "ember-source": "^4.8.0 || ^5.0.0"
+    "ember-source": "^4.8.0 || ^5.0.0",
+    "tracked-built-ins": "^3.3.0"
```

Comment:
ember-file-upload v8 lists it as a peerDependency so we do the same. New Ember projects will have this as part of the blueprint.